docs/source/rule-testing.rst

```diff
@@ -1,13 +1,13 @@
 Rule Testing
 ============
 
-In order to test the effectiveness of new rules, local integration tests can be used to verify that alerts would be triggered given a certain input.  The ``stream_alert_cli.py`` command line tool comes built-in with a ``lambda test`` command which does exactly this.
+To test the accuracy of new rules, local tests can be written to verify that alerts trigger against valid input.  The ``stream_alert_cli.py`` CLI tool comes built-in with a ``lambda test`` command which does exactly this.
 
 Configuration
 ~~~~~~~~~~~~~
 
 To test a new rule, first create a new file under ``test/integration/rules`` named ``rule_name_goes_here.json``.  This file should contain this exact structure::
```

Comment:
`RULE_NAME_GOES_HERE.json`. For example, if you wanted to test `def production_sudo(record): ...`, you'd create the file `test/integration/rules/production_sudo.json`